src/generated/CPACSPostFailure.cpp

```diff
@@ -194,7 +194,30 @@ namespace tigl
         
         bool CPACSPostFailure::ValidateChoices() const
         {
-            return ((m_initialEquivalentShearStrain_choice2.is_initialized() && m_intermediateEquivalentShearStrain_choice2.is_initialized() && m_ultimateEquivalentShearStrain_choice2.is_initialized() && m_intermediateDamage_choice2.is_initialized() && m_ultimateDamage_choice2.is_initialized()) == 1);
+            return
+            (
+                (
+                    (
+                        true // m_plasticEliminationStrain_choice1 is optional in choice
```

Comment:
I'll note it. As the choice validation is still not perfect (does allow the presence of a complete choice and elements from other incomplete choices), i might fix this in the future.